docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3"
+services:
+  nginx:
+    build:
+      context: ./
+      dockerfile: docker/Nginx/Dockerfile
+    ports: 
+      - "8080:80"
+    networks:
+      - internal
+    volumes:
+      - ./:/var/www/html/
+      - ./logs/nginx:/var/log/nginx/
+    container_name: nginx
+  php:
+    build:
+      context: .
+      dockerfile: docker/PHP/Dockerfile
+    networks:
+      - internal
+    volumes:
+      - ./:/var/www/html/
+      - ./logs/php.log:/var/log/fpm-php.www.log
+    container_name: php
+    expose:
+      - "9000"
+  mysql:
+    container_name: db
+    restart: always
+    image: mysql:latest
+    ports:
+     - "3306:3306"
+    environment:
+      MYSQL_ROOT_PASSWORD: 'password'
+      MYSQL_USER: 'root'
+      MYSQL_PASS: 'password'
+      MYSQL_DATABASE: 'dot_lms'
+    volumes:
+      - ./db-data:/var/lib/mysql
+    networks:
+      - internal
+networks:
+  internal:
```

Comment:
better to use a meaningful network name. I would prefer naming the network "dot-lms-backend" instead of "internal".